### PR TITLE
Removed the space in 'Mark Text' (#2763)

### LINKED ISSUE #2763: MarkText is a better name
### Describe your feature request

MarkText is better than Mark Text. Can you change it for all the platforms?

### Additional context [optional]

Like MacVim, MacDown, remove the space is more readable, clean and professional. It's the same as MarkDown but just change a word to define a new powerful tool.

These names could be changed:

- Official site title
- Official site name
- Desktop app name
- Desktop app alt name

### PR DESCRIPTION
**Pull request by bolshoytoster**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | no
| Breaking changes? | As far as I've tested, no
| Deprecations?     | no
| New tests added?  | not needed
| Fixed tickets     | Fixes #2763
| License           | MIT

### Description

[Description of the bug or feature]

Literally just replaced all occurances of 'Mark Text' with 'MarkText' using sed.

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ v0.15.1 is an unplanned release to fix a XSS security vulnerability.
 - Added portable zip archive for both x86 and x64 Windows.
 - Changed `viewToggleFullScreen` and `windowCloseWindow` key bindings to `windowToggleFullScreen` and `fileCloseWindow`.
 - Removed `viewChangeFont` key binding.
-- Mark Text is now single-instance application on Linux and Windows too.
```
Comment (Ovilia) [inline .github/CHANGELOG.md:191]: I think changelog should stay unchanged.

Comment (bolshoytoster) [reply .github/CHANGELOG.md:191]: Fair enough I'll revert that

Comment (Ovilia) [reply .github/CHANGELOG.md:191]: I think changelog should stay unchanged. What is released is released.

**Comment by bolshoytoster** [comment]
I've reverted the changes in the `.github` directory, including the changelogs.

I've also added an alias `cs` for `csharp`, this should resolve #2760.

**Comment by bolshoytoster** [comment]
I've now added an error handler in the render process.

This should resolve #2758

The `src/muya/lib/contentState/clickCtrl.js` file will now return if `document.querySelector` fails to find an element, instead of running into a `TypeError`.

File: src/muya/lib/contentState/clickCtrl.js [hunk 2]
```diff
@@ -12,6 +12,9 @@ const clickCtrl = ContentState => {
       const lastBlock = this.getLastBlock()
       const archor = this.findOutMostBlock(lastBlock)
       const archorParagraph = document.querySelector(`#${archor.key}`)
+      if (archorParagraph === null) {
```
Comment (Jocs) [inline src/muya/lib/contentState/clickCtrl.js:15]: Why add these codes?

**Review by Jocs — APPROVED** [review]

File: src/muya/lib/contentState/clickCtrl.js (hunk 2 above)
Comment (bolshoytoster) [reply src/muya/lib/contentState/clickCtrl.js:15]: I added this due to a TypeError that can be raised (pointed out in #2758) when `document.querySelector` returns null.

Comment (bolshoytoster) [reply src/muya/lib/contentState/clickCtrl.js:15]: I could change it to just `if (!archorParagraph) {`

**Comment by Jocs** [comment]
@bolshoytoster Thanks for your contribution.

**Pull request merged**